docs/faq/terminology.rst

```diff
@@ -0,0 +1,43 @@
+.. _Observation_Terminology:
+
+***********
+Terminology
+***********
+
+Scheduling
+==========
+
+Scheduling observations is a common task in astronomy, but there is a lack of
+formal terminology. Here we define the terminology as it will be used in astroplan
+for scheduling tasks. This is for consistency within the code and the documentation.
+
+The terms used are as follows:
+
+* **visit**: a period of time spent observing a single target
+* **observing block** (OB): the minimum schedulable block (tentative: a group of visits)
+* **priority**: number assigned to the **observing block** by the user that 
+  defines its precedence within the set of blocks to be scheduled. Should probably
+  also be on a 0->1 (0=no good, 1=good) scale, or rescaled within the scheduler
+* **rank**: a **priority** defined by a TAC rather than a user
+* **constraint**: sets limits and can yield **scores** based on inputs. Scores can be 
+  boolean or floats (0=no good, 1=good), with a flag in the ``Constraint`` call 
+  that selects which is used
+* **score**: the value returned by evaluating a **constraint**
+* **scorekeeper**: assigns a cumulative score to the OB based on some function that 
+  would be applied to all the individual **scores** (result should be in 0, 1)
+* **scheduler**: the entity that consumes the results of the **scorekeeper** and the 
+  **observing blocks** and produces a schedule
+* **Weights** (“user defined”?): preferences for which constraint's **scores** matter most 
+  (i.e. I care more about getting dark time than getting a low airmass)
+
+
+The different schedulers that are already implemented (none are implemented yet) 
```

Comment:
This should be rephrased a bit - just say straight-up what the present situation is - that these are not implemented, and this is a *tentative* set of possible schedulers.  Right now it doesn't make sense because it says "that are already implemented (none are implemented yet)".

Sorry if this seems picky, but in my experience it's best to be as accurate as possible for documentation because it tends to not get updated as promptly as code.  And users often get *very* confused if the documentation doesn't match the code.